pkg/virt-controller/watch/replicaset.go

```diff
@@ -519,35 +531,22 @@ func (c *VMReplicaSet) updateStatus(rs *virtv1.VirtualMachineReplicaSet, vms []v
 	// check if we need to update because of appeared or disappeard errors
 	errorsMatch := scaleErr == nil && c.getCondition(rs, virtv1.VMReplicaSetReplicaFailure) == nil || scaleErr != nil && c.getCondition(rs, virtv1.VMReplicaSetReplicaFailure) != nil
 
+	// check if we need to update because pause was modified
+	pausedMatch := rs.Spec.Paused == false && c.getCondition(rs, virtv1.VMReplicaSetReplicaPaused) == nil || rs.Spec.Paused == true && c.getCondition(rs, virtv1.VMReplicaSetReplicaPaused) != nil
```

Comment:
getCondition() is only ever used in a boolean fashion. Can we just change this function to 'hasCondition()' and return a boolean since that's its only use case?